modules/fastq/templates/minimap2_align_paired_end.sh

```diff
@@ -2,20 +2,18 @@
 set -euo pipefail
 
 align() {
-       local args=()
+    local args=()
     args+=("-t" "!{task.cpus}")
     args+=("-a")
+    args+=("-R" "@RG\tID:$(basename !{fastqR1})\tPL:!{platform}\tLB:!{sampleId}\tSM:!{sampleId}")
```

Comment:
```suggestion
    # MarkDuplicates uses the LB (= DNA preparation library identifier) field to determine which read groups might contain molecular duplicates, in case the same DNA library was sequenced on multiple lanes.
    args+=("-R" "@RG\tID:$(basename !{fastqR1})\tPL:!{platform}\tLB:!{sampleId}\tSM:!{sampleId}")
```